tagging.pb.go

```diff
@@ -27,9 +27,9 @@ const _ = proto.GoGoProtoPackageIsVersion3 // please upgrade the proto package
 
 // TagBatchPartSender holds metadata about the sender
 type TagBatchPartSender struct {
-	ServiceName     string `protobuf:"bytes,1,opt,name=ServiceName,json=serviceName,proto3" json:"service_name,omitempty"`
-	ServiceInstance string `protobuf:"bytes,2,opt,name=ServiceInstance,json=serviceInstance,proto3" json:"service_instance,omitempty"`
-	HostName        string `protobuf:"bytes,3,opt,name=HostName,json=hostName,proto3" json:"host_name,omitempty"`
+	ServiceName     string `protobuf:"bytes,1,opt,name=ServiceName,proto3" json:"service_name,omitempty"`
+	ServiceInstance string `protobuf:"bytes,2,opt,name=ServiceInstance,proto3" json:"service_instance,omitempty"`
+	HostName        string `protobuf:"bytes,3,opt,name=HostName,proto3" json:"host_name,omitempty"`
```

Comment:
Who knows `protoc` really well?
Are those changes breaking something?

I cannot force `buf` to generate those, maybe its just new (or old) version of `gogoslick`?

Reply:
Does it build without those json tags? Do unit tests still pass?

Should build fine without these tags, they'll just have different casing.

Reply:
I encountered this locally using the Makefile and but unfortunately I'm not sure what the cause of it is

Reply:
If we marshal these messages to json we don't want to change the case there, but I'm not sure that's the case.

That being said... I'm not even sure what the `json` thing within the `protobuf` tag does? It clearly didn't match the actual `json` tag. Do we know which one was effective?

Reply:
I would say its something "artificial" from some old versions of some lib..

Reply:
https://github.com/kentik/gohippo/blob/f4761b109d1ca9eadf4cb2776290530886618e85/tagging.proto#L9

We added those tags ourselves

Reply:
Nope.. those ones are there.. in `json` tag..
The ones that are missing are in `protobuf/json` and they are generated automatically for sure from field name (just with first letter lowercase, so we can have sometimes so strange names as `name=TTLMinutes,json=tTLMinutes`)

Reply:
Oh, I see. What a mess, LOL.

Yeah, I think those probably aren't used. We see this same thing in the tagging repo:

```
	string ServiceName = 1 [(gogoproto.jsontag) = "service_name,omitempty"];
```

generates:
```
	ServiceName     string `protobuf:"bytes,1,opt,name=ServiceName,json=serviceName,proto3" json:"service_name,omitempty"`
```

generates this JSON
```
      "sender": {
        "service_name": "transfluo-tagging - aws/next",
```